src/components/Error/Error.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
```

Comment:
It seems that 'classNames' is not used in this component. You can safely remove the import statement. 🧹